index.js

```diff
@@ -107,8 +107,12 @@ module.exports = function(el, onOver, onOut) {
     el.removeEventListener('blur', dispatchBlur, false);
   }
 
-  h.remove = function() {
+  h.remove = function(reset) {
```

Comment:
New param; if this is `true` the mouse out event will fire before removing the event listeners.